### PR TITLE
first round of codestlye for the template

### PR DESCRIPTION
**Pull request by zero-24**
First round of codestlye for the template.

### CODE REVIEW

File: templates/tauristar/index.php [hunk 1]
```diff
@@ -9,142 +9,123 @@
...
 
-			<nav class="navbar navbar-default">
+	switch ($params->get('mode', 2))
+	{
+		case 1:
+			if (!JFile::exists(JPATH_BASE . $cssFile))
+			{
+				TauristarHelper::compile();
+			}
+			$document->addStyleSheet(JUri::base() . $cssFile);
```
Comment (yvesh) [inline templates/tauristar/index.php]: Missing new line before?

**Comment by zero-24** [comment]
Fixed @yvesh but I'm not happy with that switch in general.

**Comment by yvesh** [comment]
@zero-24 thank you! Yeah me too. We iterate and improve it within time..

**Comment by zero-24** [comment]
:+1:

**Pull request merged**